strider/server.py

```diff
@@ -70,16 +71,43 @@ def custom_openapi():
     APP.openapi_schema = openapi_schema
     return APP.openapi_schema
 
-
 APP.openapi = custom_openapi
-APP.add_middleware(
-    CORSMiddleware,
+
+CORS_OPTIONS = dict(
     allow_origins=['*'],
     allow_credentials=True,
     allow_methods=["*"],
     allow_headers=["*"],
 )
 
+APP.add_middleware(
+    CORSMiddleware,
+    **CORS_OPTIONS,
+)
+
+# Custom exception handler is necessary to ensure that
+# we add CORS headers to errors and return a TRAPI response
+async def catch_exceptions_middleware(request: Request, call_next):
+    try:
+        return await call_next(request)
+    except Exception as e:
+        response = JSONResponse(
+            {"message" : {
```

Comment:
Weird whitespace here. I'd typically do this:
```suggestion
            {
                "message" : {
```